app/views/pages/privacy.html.erb

```diff
@@ -20,6 +22,8 @@
 
       <h3>Consent</h3>
       <p>By using our website, you hereby consent to our Privacy Policy and agree to its Terms and Conditions.</p>
+
+      <br>
```

Comment:
It's a little untidy from a semantic point of view. `<br>` tags are explicitly for line breaks where the break has semantic significance. They are not meant to be used as spacing elements. 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br

In this case because you have both a `<p>` and a `<br>`, I think screeneaders will still encapsulate the paragraphs separately - so for that I guess it is OK. It may be that you read `<br>` as a blank line when you are parsing html and that is why this feels easier but for most developers, this is probably not the case. What trips one person up, doesn't trip another - as with all code. I hear that you have found this PR review particularly frustrating and I can understand why - so I'm going to stop there. This is not a dealbreaker at all. The comment was intended as a heads-up in case the non-standard usage was unintentional. Since it is intentional, I am happy for this to go in as it is.

I was not suggesting inline styles but an addition to the css. If paragraphs need more margin on this particular page, you can namespace as such. I also find in terms of visual readability for the view, if you use css according to semantics, then you are less likely to end up with anomalies in spacing for elements that should follow the same visual pattern. e.g. in this case, forgetting to add an `<br>` between the `<p>` for one of them etc.

Reply:
thanks for the context. i think i am agreed in principle - but sometimes it really is just one place in one page of the site that just needs one more bit of visual spacing. these pages are unique because theres nowhere else that theres just plain text dumped into a full width text area - and sure we could fix that with a bunch of complicated css rules but its such a unique thing that it would be so much faff to sort out for v little gain vs a good old fashioned br :)